build.jam

```diff
@@ -0,0 +1,32 @@
+# Copyright René Ferdinand Rivera Morell 2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+import option ;
+import path ;
+
+local DIST_DIR = [ option.get distdir ] ;
+DIST_DIR ?= [ option.get build-dir ] ;
+DIST_DIR ?= [ path.join $(BOOST_ROOT) dist ] ;
+DIST_DIR ?= dist ;
+DIST_DIR = [ path.root [ path.make $(DIST_DIR) ] [ path.pwd ] ] ;
+
+project /boost/boostbook ;
+
+local patterns = *.dtd *.xml *.xsl LICENSE ;
+local dirs = dtd xsl ;
+
+explicit
+    [ install dist-share
+        : [ glob $(dirs)/$(patterns) $(dirs)/*/$(patterns)
+            $(dirs)/*/*/$(patterns) ]
```

Comment:
This should probably be `glob-tree`, shouldn't it? I.e. it should glob recursively.

Reply:
I think, it's better to change this to avoid surprises in the future. I'm not even sure `patterns` are needed either. Just install everything there is.

Reply:
> But the patterns are needed in the glob functions b2 has.

I meant the patterns need not be specific and should pick all files in the tree.

Reply:
I see. Let the patterns be for now then.

> Ostensibly the install and glob could be smarter about this.

Yeah, that would be nice. I'm sure it'll bite someone one day.